### ISSUE TITLE
convert -H error 

### ISSUE THREAD

**Issue by lraven** [samtools/bcftools#274]
Hi, Im using the develop branch to convert Phase3 1000 Genomes to VCF. Took a while so i figured it is best to note these issues for others. To successfully convert i had to: 

1. Remove sample file header (program was recognising it as an additional sample)
2. Replace rsIDs with chromosome (will be troublesome down the track when combining with array data)
3. Rename indels in TYPE column with sed: 
"sed 's/:ME:SVA//g' | sed 's/:ME:ALU//g' | sed 's/:ME:LINE1//g' | sed 's/:MT//g'"

> ~/bin/bcftools-develop/bcftools convert -H 1000GP_Phase3_chr22.hap,tmp/1000GP_Phase3_chr22_ins.legend,tmp/1000GP_Phase3.sample -o tmp/1000GP_Phase3_chr22.vcf --include TYPE="Biallelic_SNP" -O v

Thanks,
Lesley

**Comment by vforget** [comment]
I can confirm this bug, at least for point #1 i.e. sample header causing conversion to fail.

**Comment by wkretzsch** [comment]
Hi @lraven, thanks for the bug report!  What commits of htslib and bcftools were you using?

**Comment by wkretzsch** [comment]
#### Point 1
I can confirm this bug using the bcftools 1.2 and htslib 1.2.1 releases, as well as the current develop branch:
````
bcftools 1.2-133-ge63a37f
Using htslib 1.2.1-204-g8197cfd
````

I'll see about fixing this.

#### Point 2
This is not really a bug.  I think what @lraven is referring to is that the first column of the 1000 Genomes Phase 3 haplotypes at https://mathgen.stats.ox.ac.uk/impute/1000GP_Phase3.html is in the format:

````
CHROM:(POS|RSID):REF:ALT
````
, whereas bcftools expects the first column to be in the format:
````
CHROM:POS_REF_ALT
````

#### Point 3

@lraven: I can't get the `--include` option to work with `bcftools 1.2` or the `develop` branch.  This is with a type column containing only `Biallelic_SNP` or `Biallelic_INDEL`.  Are you sure this filter is working for you?

**Comment by wkretzsch** [comment]
Ok, I have a pull request in the works to fix this issue: #325

**Comment by wkretzsch** [comment]
@mcshane: this issue has been resolved and can be marked as such.
